pkg/monitoring/metrics/virt-handler/psi_metrics_collector.go

```diff
@@ -0,0 +1,259 @@
+package virt_handler
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	afero "github.com/spf13/afero"
+
+	k6tv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
+)
+
+const (
+	// args: QOS class, QOS class, pod UID (with _ instead of -)
+	pathFormat = "/node/sys/fs/cgroup/kubepods.slice/kubepods-%s.slice/kubepods-%s-pod%s.slice/"
+
+	// PSI metrics
+	PSIMemoryPressure = "memory.pressure"
+	PSICpuPressure    = "cpu.pressure"
+	PSIIoPressure     = "io.pressure"
+)
+
+var (
+	PSIMetricsCollector = operatormetrics.Collector{
+		Metrics: []operatormetrics.Metric{
+			psiMemoryPressureMetric,
+			psiCpuPressureMetric,
+			psiIoPressureMetric,
```

Comment:
just a question how these metrics will act when the vmi is not running?

Reply:
it tracks the files matching `"/node/sys/fs/cgroup/kubepods.slice/kubepods-%s.slice/kubepods-%s-pod%s.slice/"` in the node

no VM pod, no file